packages/terra-clinical-data-grid/tests/jest/subcomponents/__snapshots__/HeaderCell.test.jsx.snap

```diff
@@ -32,7 +32,6 @@ exports[`HeaderCell Snapshot Tests correctly applies the theme context className
       >
         <div
           className="content"
-          role="button"
```

Comment:
The title says: "Update Cell to have role undefined for empty cell", so shouldn't this role of buttons till apply?

Reply:
Updated the title.